index.html

```diff
@@ -6,12 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=inter">      
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=inter">   
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Inter?query=5.7.0/css/font-awesome.min.css">     
+    <script defer src="popUp.js"></script>
```

Comment:
- [ ] [REQUIRED] Kindly, update the source link for the script file and make sure it matches exactly the name of your file (you used `popUp.js` while the original name of the file is `popup.js`). Because the js script is not working.